index.html

```diff
@@ -4,18 +4,97 @@
   <meta charset="UTF-8">
   <title>Does the CSSU have a microwave?</title>
   <style>
-    h1 {
+    #yes {
       font-size: 6em;
       font-weight: 700;
       margin: 0;
       position: absolute;
       top: 50%;
       left: 50%;
       transform: translate(-50%, -50%);
+      text-align: center;
+    }
+    #since{
+      margin: 0 auto;
+      text-align: center;
+      font-size: 0.15em;
     }
   </style>
 </head>
 <body>
-  <h1>YES.</h1>
+  <div id="yes">
+    YES.
+    <div id="since"></div>
+  </div>
+  <script>
+    //TODO: update this on commit automatically
+    var LAST_UPDATE = "Thu Jan 05 2017 10:00:54 GMT-0500 (EST)";
+
+    function getTimeSince(starttime){
+      let t = Date.parse(new Date()) - Date.parse(starttime);
+      let seconds = Math.floor( (t/1000) % 60 );
+      let minutes = Math.floor( (t/1000/60) % 60 );
+      let hours = Math.floor( (t/(1000*60*60)) % 24 );
+      let days = Math.floor( t/(1000*60*60*24) );
+      return {
+        'total': t,
+        'days': days,
+        'hours': hours,
+        'minutes': minutes,
+        'seconds': seconds
+      };
+    }
+
+    function updateTimer(){
+      var time = getTimeSince(LAST_UPDATE);
+      var sinceEl = document.getElementById("since");
```

Comment:
Also, grab the element reference outside of this function so it doesn't repeatedly do it.